cognee/infrastructure/llm/gemini/adapter.py

```
@@ -0,0 +1,140 @@
+from typing import Type, Optional
+from pydantic import BaseModel
+import json
+import logging
+from litellm import acompletion, JSONSchemaValidationError
+import litellm
+from litellm import acompletion, JSONSchemaValidationError
```

Ruff (0.8.2)

Redefinition of unused `acompletion` from line 5

Remove definition: `acompletion`

(F811)